STX.REST.RESTFulSense.Clients/Services/Foundations/HttpExchanges/HttpExchangeService.cs

```diff
@@ -40,19 +42,35 @@ internal HttpExchangeService(IHttpBroker httpClientBroker)
                 httpExchange,
                 httpResponseMessage);
         });
-
+        
         private static HttpRequestMessage MapToHttpRequest(
-            HttpExchangeRequest httpExchangeRequest)
+            HttpExchangeRequest httpExchangeRequest,
+            HttpMethod defaultHttpMethod)
         {
             Uri baseAddress = new Uri(httpExchangeRequest.BaseAddress);
```

Comment:
var baseAddress =
because type is on right side

Reply:
Treated